packages/monaco/src/browser/monaco-diff-editor.ts

```diff
@@ -48,6 +48,15 @@ export class MonacoDiffEditor extends MonacoEditor {
         this._diffEditor.setModel({ original, modified });
     }
 
+    get targetUri(): URI | undefined {
+        if (this.modifiedModel.uri.startsWith('file://')) {
```

Comment:
`this.diffEditor.getModifiedEditor().getModel().uri.scheme === 'file'` - it should rely on a parsed representation

Reply:
Agree. That better. Thank you